modules/seqexec/server/src/main/scala/seqexec/server/ODBSequencesLoader.scala

```diff
@@ -8,7 +8,7 @@ import cats.Endo
 import cats.MonadError
 import cats.effect.Concurrent
 import cats.effect.Timer
-import cats.syntax.all._
+import cats.implicits._
```

Comment:
Why isn't `cats.syntax.all_` enough here?

Reply:
Actually I took another route and it required the change and I forgot to revert it.
At any rate I'm tending to use the `cats.implicits._` import